Makefile

```diff
@@ -434,8 +434,8 @@ endef
 
 define docker-test-build-push
 	docker build . -t k8gb:$(COMMIT_HASH)
-	docker tag k8gb:$(COMMIT_HASH) $(REPO):v$(COMMIT_HASH)
-	docker push $(REPO):v$(COMMIT_HASH)
+	docker tag k8gb:$(COMMIT_HASH) $(REPO):$(COMMIT_HASH)
+	docker push $(REPO):$(COMMIT_HASH)
```

Comment:
Although without `v` it is valid semver, wouldn't we keep it for consistency? (we use prefix `v` everywhere)

Reply:
in this specific case `v` is getting mixed with short hash and looks like `vc105f41` instead of `c105f41` which is quite misleading.
In any case it
* was broken for local setup
* exists for local testing only

Reply:
true 👍